ClinGen/clincoded
# Connects to #1491. Updated psycopg2 from version 2.6 to 2.7.3.1

**Steps to test:**
1. Deploy a normal test instance on AWS.
2. Check the `cloud-init-output.log` file in the test instance.
3. Expect to see the `psycopg2` package being installed without error:
```
Getting distribution for 'psycopg2==2.7.3.1'.
psycopg2.tests.__pycache__.test_module.cpython-34: module references __file__
psycopg2.tests.__pycache__.test_types_basic.cpython-34: module references __file__
psycopg2.__pycache__._psycopg.cpython-34: module references __file__
Got psycopg2 2.7.3.1.
```
